frontend/src/hooks/useRecordAudio.js

```diff
@@ -78,6 +78,7 @@ const useRecordAudio = (
 
   return {
     transcribing,
+    transcriptText: transcript?.text,
```

Comment:
this is a new property that was surfaced to upper components in order to render in alert component